pkg/test/kube/util.go

```diff
@@ -86,6 +86,8 @@ func NewPodMustFetch(a istioKube.CLIClient, namespace string, selectors ...strin
 	}
 }
 
+var ErrNoPodsFetched = fmt.Errorf("no pods fetched")
```

Comment:
Why don't you add it to the var block at the top of this file?

Reply:
Fixed.